packages/shared/src/hooks/useFeedLayout.ts

```diff
@@ -111,7 +114,13 @@ export const useFeedLayout = ({
   const { insaneMode } = useContext(SettingsContext);
   const { isSearchPageLaptop } = useSearchResultsLayout();
 
-  const isListMode = isSearchPageLaptop || insaneMode;
+  const { value: isSearchListModeExperiment, isLoading: isLoadingExperiment } =
+    useConditionalFeature({
+      feature: feature.searchListMode,
+      shouldEvaluate: isSearchPageLaptop,
+    });
+
+  const isListMode = isSearchListModeExperiment || insaneMode;
```

Comment:
why not do this in the search results layout, this way we don't have to export anything like loading experiment from here?